src/coreclr/jit/emitarm64.cpp

```diff
@@ -2626,6 +2626,48 @@ void emitter::emitInsSanityCheck(instrDesc* id)
             assert(isScalableVectorSize(elemsize));
             break;
 
+        case IF_SVE_BX_2A: // ...........ixxxx ......nnnnnddddd -- sve_int_perm_dupq_i
+            imm      = emitGetInsSC(id);
+            elemsize = id->idOpSize();
+            assert(insOptsScalableStandard(id->idInsOpt()));
+            assert(isVectorRegister(id->idReg1()));
+            assert(isVectorRegister(id->idReg2()));
+            assert(isScalableVectorSize(elemsize));
+#ifdef DEBUG
```

Comment:
don't think you need `#ifdef DEBUG` here. All these has is `assert` and compiler should be able to eliminate it.

Reply:
Yea, this has been commented about before. Personally, I like to have the `ifdef` for these situations as it distinguishes code that is for verification. Asserts on their own obviously don't need an `ifdef`, but for checks that require branching, I tend to wrap them around an `ifdef`.